### DISCUSSION TITLE
React Hook Form V6

### DISCUSSION THREAD

**Discussion by bluebill1049** [react-hook-form/react-hook-form#1783] [Show and tell] [answered]
We will likely release V6 in the next week (12th). If you have any feedback with the RC version, please let us know.

**Comment by bluebill1049** [comment, marked as answer]
Sorry for the delay. We try to deliver a solid version 6 and not rushing it.

**Reply by stunaz** [reply]
What's new on V6? is there a get started docs with v6?

**Reply by zelucena** [reply]
Please watch this video posted by @bluebill1049 
https://www.youtube.com/watch?time_continue=12&v=CeR7ERx9ehE&feature=emb_logo

**Comment by bluebill1049** [comment]
Made a short introduction video about @HookForm v6 before the final release. If you are planning to migrate or interested, feel free to leave feedback at Github discussion.

https://www.youtube.com/watch?time_continue=12&v=CeR7ERx9ehE&feature=emb_logo

**Reply by zelucena** [reply]
`shouldUnregister: false` seems to be exactly what I was looking for yesterday. I'm looking forward to migrate asap
